context/app/static/js/components/entity-search/sidebar/DatasetConfigureSearchTable/hooks.js

```diff
@@ -25,21 +24,24 @@ function selectedItemsReducer(state, { type, payload }) {
 }
 
 function useGroupedFieldConfigs() {
-  const { initialFields, initialFacets } = useStore();
-  const metadataFieldConfigs = useMetadataFieldConfigs();
-
-  // Order matters. Field configs defined in initialFields/initialFacets will be overwritten.
-  const allFieldConfigs = { ...initialFields, ...initialFacets, ...metadataFieldConfigs };
-  return Object.entries(allFieldConfigs).reduce((acc, [metadataFieldName, metadataFieldConfig]) => {
-    return produce(acc, (draft) => {
-      // eslint-disable-next-line no-param-reassign
-      draft[metadataFieldConfig.configureGroup] = {
-        ...draft[metadataFieldConfig.configureGroup],
-        [metadataFieldName]: metadataFieldConfig,
-      };
-      return draft;
-    });
-  }, {});
+  const { availableFields } = useStore();
+
+  const groupedFieldConfigs = Object.entries(availableFields).reduce(
+    (acc, [metadataFieldName, metadataFieldConfig]) => {
+      return produce(acc, (draft) => {
+        // eslint-disable-next-line no-param-reassign
+        draft[metadataFieldConfig.configureGroup] = {
+          ...draft[metadataFieldConfig.configureGroup],
+          [metadataFieldName]: metadataFieldConfig,
+        };
+        return draft;
+      });
+    },
+    {},
+  );
+  // Return only sample and donor metadata
+  const { 'Dataset Metadata': datasetMetadata, ...rest } = groupedFieldConfigs;
+  return rest;
```

Comment:
```suggestion
  delete groupedFieldConfigs.datasetMetadata;
  return groupedFieldConfigs;
```
Would this do the same thing? But whatever you prefer.

Reply:
I opted for this at first, but it was erroring out for a reason I've now forgotten.